spec/accountSpec.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var Account = require('../src/account').Account
+var validateDate = require('../src/account').validateDate
+var convertStringToDate = require('../src/transaction').convertStringToDate
+
+describe('Account', function () {
+  var account, date, transaction, dateConverted;
+  beforeEach(function () {
+    date = '10/01/2012'
+    dateConverted = convertStringToDate(date)
+    account = new Account();
+  });
+
+  describe('is initialized', function () {
+    it('with an empty array of transactions', function () {
+      expect(account.transactions).not.toBe(null);
```

Comment:
Is there a way to check that the account starts with no transactions by testing account's behaviour, rather than its state?